frontend.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>

```

Comment:
Please set HTML tabs to 4 spaces in your text editor. 8 takes up too much space :)